tests/toeplitz.sh

```diff
@@ -29,8 +33,12 @@
 IPVER=-6
 PROTO=-u
 DPORT=8000
+DEV=""
+IRQ_PREFIX=""
+TEST_RSS=false
```

Comment:
Wait a sec, I don't think this is necessary since the presence of a specified IRQ_PREFIX means "test RSS," so this TEST_RSS var is superfluous. I'll fix this real quick and push once more.

Reply:
Done. PTAL.